src/proxy-ui-api/frontend/src/store/modules/addClient.ts

```diff
@@ -112,12 +112,33 @@ export const actions: ActionTree<AddClientState, RootState> = {
       });
   },
 
+  fetchSelectableForSubsystem({ commit, rootGetters }, client: Client) {
+    // Fetch clients from backend that match the selected client without subsystem code
+    return api.get(`/clients?instance=${client.instance_id}&member_class=${client.member_class}&member_code=${client.member_code}&internal_search=false&show_members=false&is_not_local_client=true`)
```

Comment:
There are a couple of changes to queryparameters in develop so I guess this change should be made: 
is_not_local_client -> exclude_local

Reply:
Fixed